File: Backend/fuel-app-server/src/main/java/com/dimitriskikidis/fuelappserver/fuelstation/FuelStationCreateUpdateRequest.java
package com.dimitriskikidis.fuelappserver.fuelstation;

public record FuelStationCreateUpdateRequest(
        Integer brandId,

        Double latitude,

        Double longitude,

        String name,
        String city,

        String address,
        String postalCode,

        String phoneNumber
) {
}

File: Android/FuelAppforOwners/app/src/main/java/com/dimitriskikidis/owner/fuelapp/data/remote/requests/FuelStationCreateUpdateRequest.kt
package com.dimitriskikidis.owner.fuelapp.data.remote.requests

data class FuelStationCreateUpdateRequest(
    val brandId: Int,
    val latitude: Double,
    val longitude: Double,
    val name: String,
    val city: String,
    val address: String,
    val postalCode: String,
    val phoneNumber: String
)
